crates/aptos/src/common/init.rs

```diff
@@ -203,17 +203,14 @@ impl CliCommand<()> for InitTool {
                     "Account {} doesn't exist, creating it and funding it with {} Octas",
                     address, NUM_DEFAULT_OCTAS
                 );
-                match fund_account(
+                fund_account(
                     Url::parse(faucet_url)
                         .map_err(|err| CliError::UnableToParse("rest_url", err.to_string()))?,
                     NUM_DEFAULT_OCTAS,
                     address,
                 )
-                .await
-                {
-                    Ok(_) => eprintln!("Account {} funded successfully", address),
-                    Err(err) => eprintln!("Account {} failed to be funded: {:?}", address, err),
```

Comment:
This does change the experience a bit, since you'll have to type in your private key again in the event of a faucet failure.  It used to do this, idk when it stopped doing that

Reply:
Better this way still I think, I'd rather it fail and I go through the full flow again than exit in some half way state.